packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -171,6 +176,8 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         {...this.props}
         chartTitle={this._getChartTitle()}
         points={this._points}
+        yMinValue={this.props.supportNegativeData ? this._yMin : 0}
```

Comment:
>0 [](http://example.com/codeflow?start=65&length=1)

dont hardcode this to 0.

Reply:
We need to pass this._yMin only for negative case. For other case, we were not passing yMin earlier.

Reply:
We don't need this. Already being passed in spread operator.